dce_course_info/requirements/aws.txt

```diff
@@ -14,4 +14,4 @@ djrill==1.3.0
 #django-postgrespool==0.3.0
```

Comment:
-- Managed to get rid of .env file and its surrounding requirements
-- postgrespool and database-url were unused after using TLT database configuration settings
-- gunicorn is not going to be used in TLT AWS